libopae/CMakeLists.txt

```diff
@@ -69,6 +72,13 @@ target_link_libraries(opae-c
   ${libuuid_LIBRARIES}
   ${CMAKE_THREAD_LIBS_INIT})
 
+if (USE_NUMA)
+    find_package(NUMA REQUIRED)
```

Comment:
If numactl is not present, will this fail the entire build?

Reply:
Yes.  If you have set the option to use NUMA and it's not present, the build will fail.  Is this an issue?

Reply:
Ah, no. If USE_NUMA is not set by default, and someone sets it, no NUMA package should fail the build.